src/openrct2/world/Location.hpp

```diff
@@ -614,35 +614,35 @@ struct TileCoordsXYZD : public TileCoordsXYZ
  */
 template<class T> struct CoordsRange
 {
-    T LeftTop{ 0, 0 };
-    T RightBottom{ 0, 0 };
+    T PointOne{ 0, 0 };
+    T PointTwo{ 0, 0 };
 
-    int32_t GetLeft() const
+    int32_t GetXOne() const
```

Comment:
```suggestion
    int32_t GetX1() const
```

and the same for the other three.